zips/zip-1015.rst

```diff
@@ -203,16 +207,19 @@ The funds SHALL be received and administered by the Financial Privacy Foundation
 administration of the ZCG program, but subject to the following additional
 constraints:
 
-1. These funds MUST only be used to issue grants to external parties that are
-   independent of FPF, and to pay for expenses reasonably related to the
-   administration of the ZCG program. They MUST NOT be used by FPF for
-   its internal operations and direct expenses not related to the
-   administration of grants or the grants program.
+1. These funds MUST only be used to issue Zcash Community Grants to external
+   parties that are independent of FPF or to Autonomous Entities that operate
+   under the FPF umbrella, and to pay for expenses reasonably related to
+   the administration of Zcash Community Grants. They MUST NOT be used by FPF
+   for its internal operations and direct expenses not related to
```

Comment:
```suggestion
   for its internal operations or for direct expenses not related to
```

Reply:
Does this resolve the issue? These funds must be used exclusively for issuing Zcash Community Grants to external parties that are independent of the FPF or to Autonomous Entities operating under the FPF umbrella. Additionally, they may be used to cover expenses reasonably related to the administration of Zcash Community Grants. Under no circumstances should these funds be used by FPF for its internal operations or for any direct expenses unrelated to the administration of Zcash Community Grants.